python/idsse_common/test/test_aws_utils.py

```diff
@@ -2,63 +2,79 @@
 # --------------------------------------------------------------------------------
 # Created on Wed Jun 21 2023
 #
-# Copyright (c) 2023 Colorado State University. All rights reserved.
+# Copyright (c) 2023 Colorado State University. All rights reserved. (1)
 #
 # Contributors:
-#     Mackenzie Grimes
+#     Mackenzie Grimes (1)
```

Comment:
You added the `(1)` annotation to CSU contribution, did you want to also add CU and your name to licensing here?